wsagent/che-core-api-git/src/main/java/org/eclipse/che/api/git/GitConflictException.java

```diff
@@ -0,0 +1,52 @@
+/*******************************************************************************
+ * Copyright (c) 2012-2015 Codenvy, S.A.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *   Codenvy, S.A. - initial API and implementation
+ *******************************************************************************/
+
+package org.eclipse.che.api.git;
+
+import org.eclipse.che.dto.shared.DTO;
+
+import java.util.LinkedList;
+import java.util.List;
+
+/**
+ * Created by I053322 on 7/7/2016.
+ */
+@DTO
```

Comment:
If I am not mistaken this exception is used only in server side so @DTO annotation is not needed here